src/go/rpk/pkg/cli/cluster/config/config.go

```diff
@@ -48,6 +48,7 @@ different redpanda version that does not recognize certain properties.`,
 		"Include all properties, including tunables",
 	)
 	p.InstallAdminFlags(cmd)
+	p.InstallSASLFlags(cmd)
```

Comment:
should the ordering be consistent on these?

Reply:
haha yeah, good catch.